sdk/orm-django/django_encryption/fields.py

```diff
@@ -90,6 +249,69 @@ def vault_collection(self):
         return vault_collection
 
     def from_db_value(self, value, *args, **kwargs):
```

Comment:
I can't see any use of [from_db_value, __ from_db_value] 
If it something that called by djungo internal, lets add comments to describe it

Reply:
Added comment.
This is indeed a django function.

Reply:
can you add also for : __from_db_value?

Reply:
Fixed (removed the function)